### PR TITLE
Replace legacy common/log with promlog

### LINKED ISSUE #62: Switch logging to promlog
Similar to the other official Prometheus projects, we have been switching logging from `github.com/prometheus/common/log` to `github.com/prometheus/common/promlog`. This uses go-kit for logging.

### PR DESCRIPTION
**Pull request by dswarbrick**
This is a fairly naive replacement of the legacy prometheus/common/log wrapper with promlog. Fixes #62 

@SuperQ

### CODE REVIEW

File: bind_exporter.go [hunk 1]
```diff
@@ -443,7 +446,7 @@ func (s *statisticGroups) String() string {
 	for _, g := range *s {
 		groups = append(groups, string(g))
 	}
-	return fmt.Sprintf("%q", strings.Join(groups, ","))
+	return strings.Join(groups, ",")
```
Comment (SuperQ) [inline bind_exporter.go:449]: Hmm, the `fmt.Sprintf()` bit is for string quoting safety. I wonder why it was like that.

Comment (dswarbrick) [reply bind_exporter.go:449]: If we leave it as it was, go-kit logs it double-quoted, e.g. `collectors="\"foo,bar,bz\""`.

Comment (SuperQ) [reply bind_exporter.go:449]: Ahh, ok, that makes sense.

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**